### ISSUE TITLE
Accessibility: MAS20B: Link Purpose: Name property is not descriptive for the ".tar.gz" link

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#26]
#A11yMAS
#MAS20B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
**Scenario-1:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in EDGE.
2. Navigate to ".tar.gz" link (with mouse as keyboard is not accessible) available under the Debian, Ubuntu download link inside "Get Started Now" header
3. Observe the Name property in Inspect

**Same issue is also observed with the ".zip" link available under "Windows" download link inside "Get Started Now" header**
**_Same issue is also observed with the links:_**
[https://code.visualstudio.com/Download
](https://code.visualstudio.com/Download
)https://code.visualstudio.com/insiders

**Scenario-2:**
The same issue is observed with the Windows, .deb, .rpm, Mac download links avilable  in the page under "Get Started Now" header
**_Same issue is also observed with the links:_**
https://code.visualstudio.com/Download
https://code.visualstudio.com/insiders

**Actual Result:**
**Scenario-1:**
The Name property is not descriptive
**_Narrator Behaviour:_**
Narrator announces as ".tar.gz"

**Scenario-2:**
The Name property is not descriptive
**_Narrator Behaviour:_**
Windows Windows 7, 8, 10 button
.deb Debian, Ubuntu button
.rpm Red Hat, Fedora, CentOS button
Mac macOS 10.X button

**Expected Result:**
The Name property should be descriptive so that the users able to understand what is the purpose of the link
**_Suggested Fix:_**
Either "Name" property should be descriptive or "Help Text" property should be added

**User Impact:**
The visually impaired users will not be able to understand the exact purpose of the link

**Screenshots:**
![default tar gz name property is not descriptive](https://cloud.githubusercontent.com/assets/24825441/21709437/00d08a36-d407-11e6-8cf1-ecb3d14266e6.png)
![name property is not descriptive for zip link](https://cloud.githubusercontent.com/assets/24825441/21709436/00d019c0-d407-11e6-825a-b2135f44c977.png)
![name property is not descriptive for mac download link](https://cloud.githubusercontent.com/assets/24825441/21709439/00d37a16-d407-11e6-9fa5-13834236044f.png)
![name property is not descriptive for windows download link](https://cloud.githubusercontent.com/assets/24825441/21709438/00d203c0-d407-11e6-8d68-50b0f559ee2b.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/6efd7c2db5268d27dd8c690eb8a501420fe345ad

and

https://github.com/Microsoft/vscode-website/commit/e5666cde8df5932957fc3dd1af31f88d27faaee2
